site/_docs/reference.md

```diff
@@ -2294,6 +2294,7 @@ semantics.
 | C | Operator syntax                                | Description
 |:- |:-----------------------------------------------|:-----------
 | p | expr :: type                                   | Casts *expr* to *type*
+| m | BIT_COUNT(integer)                             | Returns the bitwise COUNT of non-null *integer*
```

Comment:
How about the `null` input, i.e., `bit_count(null)`? I think we'd better add some tests and update the doc as that of `BIT_XOR`.

Reply:
> How about the `null` input, i.e., `bit_count(null)`? I think we'd better add some tests and update the doc as that of `BIT_XOR`.

Need to add this as a null test? Because `defineMethod (BIT_COUNT, BuiltInMethod.BIT_COUNT.method, NullPolicy.STRICT); `are strictly defined and cannot be null.

Reply:
`NullPolicy.Strict` prevents the function call when one input is `null` and directly returns `null`. 
Therefore, I think we'd better tell end-users what he will get for `bit_count(col)` when `col` is `null`. 
E.g.,
> Returns the bitwise XOR of all non-null input values, or null if none

Reply:
+1 on what @DonnyZone said.